presto-docs/src/main/sphinx/sql/source.rst

```diff
@@ -0,0 +1,23 @@
+======
+SOURCE
+======
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SOURCE file-path
+
+Description
+-----------
+
+Execute all complete sql statements in the given file.
```

Comment:
Are there any restrictions in the commands that can go in this file? I remember that there is an issue with using "USE" with files and -f option of CLI. Please verify that and add a note to mention such things.